File: twitter/model/tweet.py
from twitter.model.twitter_user import TwitterUser

class Tweet:
    def __init__(self, tweet):
        self.text = tweet.text
        self.user = TwitterUser(tweet.user)
        self.created_at = tweet.created_at

    def get_dict_data(self):
        data = {
            "text": self.text,
            "user": self.user,
            "year": self.created_at
        }
        return data

File: src/app/models/tweet.ts
import {TwitterUser} from "./twitter-user";
import {Entity} from "./entity";

export class Tweet {
  id: number | undefined;
  text: string | undefined;
  user: TwitterUser = new TwitterUser();
  entities: Entity = new Entity();
  sentiment: string | undefined;
  created_at: Date | undefined;
}
